stable/syncthing/2.1.0/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: syncthing
+version: 2.1.0
+upstream_version: 1.15
+appVersion: "auto"
+description: P2P file synchronization application
+type: application
+deprecated: false
+home: https://syncthing.net/
+icon: https://raw.githubusercontent.com/syncthing/syncthing/main/assets/logo-128.png
+keywords:
+  - syncthing
+sources:
+  - https://github.com/syncthing/syncthing
+  - https://hub.docker.com/r/syncthing/syncthing
+dependencies:
+  - name: common
+    repository: https://truecharts.org/
+    version: 2.1.4
+    # condition:
+maintainers:
+  - name: TrueCharts
+    email: info@truecharts.org
+    url: truecharts.org
+  - name: Ornias1993
+    email: kjeld@schouten-lebbing.nl
+    url: truecharts.org
+  - name: luuknieuwdorp
+    email: luuk@nieuwdorp.me
+    url: nieuwdorp.me
```

Comment:
If you want to be maintainer for this, which would be awesome, please also add yourself to .github/CODEOWNERS